src/sentry/snuba/metrics/datasource.py

```diff
@@ -122,18 +133,18 @@ def get_metrics(projects: Sequence[Project]) -> Sequence[MetricMeta]:
             projects=projects,
             org_id=projects[0].organization_id,
         ):
-            try:
+            mri = reverse_resolve(row["metric_id"])
+            public_name = get_public_name_from_mri(mri)
+            if public_name is not None:
                 metrics_meta.append(
                     MetricMeta(
-                        name=get_public_name_from_mri(reverse_resolve(row["metric_id"])),
+                        name=public_name,
                         type=metric_type,
                         operations=AVAILABLE_OPERATIONS[METRIC_TYPE_TO_ENTITY[metric_type].value],
                         unit=None,  # snuba does not know the unit
                     )
                 )
-            except InvalidParams:
-                # An instance of `InvalidParams` exception is raised here when there is no reverse
-                # mapping from MRI to public name because of the naming change
+            else:
                 logger.error("datasource.get_metrics.get_public_name_from_mri.error", exc_info=True)
                 continue
             metric_ids_in_entities[metric_type].add(row["metric_id"])
```

Comment:
Not sure if we want to include Custom Measurements here

Reply:
I think we should keep them excluded for now? in case this function is being used somewhere that doesnt expect CM. I don't think we'll be using this function for our CM work